modules/log/src/log.ts

```diff
@@ -8,7 +8,7 @@ import {addColor} from './utils/color';
 import {autobind} from './utils/autobind';
 import assert from './utils/assert';
 import {getHiResTimestamp} from './utils/hi-res-timestamp';
-import {nodeAsciifyImage} from './node/node-asciify-image';
+import nodeAsciifyImage from './node/node-asciify-image';
```

Comment:
Nit: I've had problems with a default import on browser on some bundlers. Who knows where things stand today, but I think `import * as node` has been most reliable and I think that is what is used in loaders.gl